### PR TITLE
Esc propagation

### PR DESCRIPTION
**Pull request by drakegens**

### CODE REVIEW

File: src/Multiselect.elm [hunk 1]
```diff
@@ -214,7 +212,7 @@ update : Msg -> Model -> ( Model, Cmd Msg, Maybe OutMsg )
 update msg (Model model) =
     case msg of
         Start ->
-            (Model model, Cmd.none, Nothing )
+            ( Model model, Cmd.none, Nothing )
```
Comment (cjmeeks) [inline src/Multiselect.elm:215]: seems like he didnt run elm-format lol

Comment (stephenreddek) [reply src/Multiselect.elm:215]: yah... might want to turn off elm-format for these changes or do that separately or something. it makes the diff hard to read.

File: src/Multiselect.elm [hunk 2]
```diff
@@ -792,6 +813,27 @@ preventDefaultButtons =
     Html.Styled.Events.custom "keydown" (withOptions options decoder)
 
 
+preventPropagationButtons : Html.Styled.Attribute Msg
+preventPropagationButtons =
```
Comment (stephenreddek) [inline src/Multiselect.elm:817]: calling these buttons doesn't make sense to me

File: src/Multiselect.elm [hunk 3]
```diff
@@ -752,11 +764,20 @@ input (Model model) =
 
             else
                 Html.Styled.Attributes.property "type" (Encode.string "text")
+
+        inputDivAttribute =
```
Comment (drakegens) [inline src/Multiselect.elm:768]: @stephenreddek  here's where i'm preventing based on Opened

**Comment by drakegens** [comment]
Hot tip you can hide whitespace changes on PRs. Going to keep the elm format in.

**Pull request merged**